tox.ini

```diff
@@ -9,7 +9,7 @@ deps =
     -r{toxinidir}/docs/requirements.txt
     -r{toxinidir}/ut-requirements.txt
     flake8 >= 3.7.9
-    pylint >= 2.10.2
+    pylint >= 2.10.2,<=2.13.9
```

Comment:
Why did you need to reduce the pylint requirement? I recommend simply resolving and committing all errors you found in the newer version of pylint so we remain upstream as much as possible. I know it can be annoying but that's the approach we took so far.

Reply:
Pylint deprecated options were removed and resolved, and I removed the max version limit.
https://github.com/ctera/ctera-python-sdk/pull/156/commits/1091372d41deb0ae6fbc58d96b09d95aa73a96bd